foo/bar.js

```diff
@@ -0,0 +1,22 @@
+// This is a sample file for screenshot purposes
+
+
+export function isChicken(term: string): boolean {
+    switch (term) {
```

Comment:
praise:

Nice use of `switch()`! Makes this easy to extend to other languages.